packages/sdk/src/commons/index.ts

```diff
@@ -16,23 +16,27 @@ export enum ChainId {
 export const CACHER = new Cacher("carrot-kpi-sdk");
 
 export interface ChainAddresses {
+    multicall3: Address;
     factory: Address;
     kpiTokensManager: Address;
     oraclesManager: Address;
 }
 
 export const CHAIN_ADDRESSES: Record<ChainId, ChainAddresses> = {
     [ChainId.GNOSIS]: {
+        multicall3: "0xcA11bde05977b3631167028862bE2a173976CA11",
         factory: "0x2C9c7E6d05a98f771C22819C80D70827d2d9eBAF",
         kpiTokensManager: "0xfE0CFcbCb1746054c342aC975278CFb5D6afa33F",
         oraclesManager: "0x928d003c0D4e75fb08F4A90E46963c34514E6C42",
     },
     [ChainId.SEPOLIA]: {
+        multicall3: "0xcA11bde05977b3631167028862bE2a173976CA11",
```

Comment:
Is the `sepolia` multicall address the same as the `gnosis` one?

Reply:
Yessir. The official deployment has the same address across 70+ chains. The Scroll testnet address is different because I had to deploy the contract myself.